src/api.js

```diff
@@ -1362,9 +1364,12 @@ api.declare({
     res.once('close', accept);
   });
 
-  let result = await this.workClaimer.claim(
-    provisionerId, workerType, workerGroup, workerId, count, aborted,
-  );
+  let [result] = await Promise.all([
+    this.workClaimer.claim(
+      provisionerId, workerType, workerGroup, workerId, count, aborted,
+    ),
+    this.workerInfo.provisionerSeen(provisionerId),
```

Comment:
We could also choose to not block on `provisionerSeen`... it call it but not `await` it... we would obviously have to report errors directly to sentry then...